dataplane/unittest/vrf_lpm_common.h

```diff
@@ -0,0 +1,363 @@
+#pragma once
+
+#include <algorithm>
+
+#include "../vrf.h"
+
+namespace dataplane::vrflpm
+{
+
+inline uint32_t IpAddressToInternal(const ipv4_address_t& address)
+{
+	return rte_cpu_to_be_32(address.address);
+}
+
+inline std::array<uint8_t, 16> IpAddressToInternal(const ipv6_address_t& address)
+{
+	return common::ipv6_address_t(address.bytes);
+}
+
+inline std::string AddressStr(const ipv6_address_t& address)
+{
+	return common::ipv6_address_t(address.bytes).toString();
+}
+
+inline std::string AddressStr(const std::array<uint8_t, 16>& address)
+{
+	return common::ipv6_address_t(address).toString();
+}
+
+inline std::string AddressStr(const ipv4_address_t& address)
+{
+	return common::ipv4_address_t(address.address).toString();
+}
+
+inline std::string AddressStr(const uint32_t& address)
+{
+	return common::ipv4_address_t(address).toString();
+}
```

Comment:
Those are functions that are used only for this .h file, so they should be `static` instead of `inline`.